Source/WebKit/UIProcess/win/PageClientImpl.cpp

```diff
@@ -232,6 +240,70 @@ void PageClientImpl::preferencesDidChange()
     notImplemented();
 }
 
+bool PageClientImpl::handleRunOpenPanel(WebPageProxy*, WebFrameProxy*, const FrameInfoData&, API::OpenPanelParameters* parameters, WebOpenPanelResultListenerProxy* listener)
+{
+    ASSERT(parameters);
+    ASSERT(listener);
+
+    HWND viewWindow = viewWidget();
+    if (!IsWindow(viewWindow))
+        return false;
+
+    bool isAllowMultipleFiles = parameters->allowMultipleFiles();
+    Vector<WCHAR> fileBuf(isAllowMultipleFiles ? maxFilePathsListSize : MAX_PATH);
+
+    OPENFILENAME ofn;
+
+    memset(&ofn, 0, sizeof(ofn));
+
+    // Need to zero out the first char of fileBuf so GetOpenFileName doesn't think it's an initialization string
+    fileBuf[0] = '\0';
+
+    ofn.lStructSize = sizeof(ofn);
+    ofn.hwndOwner = viewWindow;
+    String allFiles = makeString(allFilesText(), String("\0*.*\0\0", 6));
+
+    Vector<wchar_t> filterCharacters = allFiles.wideCharacters(); // Retain buffer long enough to make the GetOpenFileName call
+    ofn.lpstrFilter = filterCharacters.data();
+
+    ofn.lpstrFile = fileBuf.data();
+    ofn.nMaxFile = fileBuf.size();
+    String dialogTitle = uploadFileText();
+    Vector<wchar_t> dialogTitleCharacters = dialogTitle.wideCharacters(); // Retain buffer long enough to make the GetOpenFileName call
```

Comment:
Use `auto`. You are mixing `wchar_t` and `WCHAR`.  I guess WebKit is using `wchar_t` more than `WCHAR`

Reply:
Fixed at 19fcde7